src/main/resources/init.sh

```diff
@@ -77,11 +77,11 @@ status)
                 exit 0
             else
                 printf "%s\n" "Warning, Pid $PID appears to not correspond to service $SERVICE"
-                exit 0
             fi
         fi
-        printf "%s\n" "Process dead but pidfile exists"
-        exit 1
+        printf "%s\n" "Process dead but pidfile exists. Cleaning up."
+        rm -f $PIDFILE
+        exit 3
```

Comment:
👎  the existing implementation is per the Linux Base Spec: https://refspecs.linuxbase.org/LSB_3.0.0/LSB-PDA/LSB-PDA/iniscrptact.html

> If the status action is requested, the init script will return the following exit status codes.
> 
> 0	program is running or service is OK
> 1	program is dead and /var/run pid file exists
> 2	program is dead and /var/lock lock file exists
> 3	program is not running
> 4	program or service status is unknown
> 5-99	reserved for future LSB use
> 100-149	reserved for distribution use
> 150-199	reserved for application use
> 200-254	reserved